config/routes.rb

```diff
@@ -610,8 +610,8 @@ def scoped_navigation_routes(context, navigation)
         get "/coming-soon", to: "state_file_pages#coming_soon"
         post "/clear_session", to: 'state_file_pages#clear_session'
         get "/privacy-policy", to: "state_file_pages#privacy_policy"
-        get "/unsubscribe_email", to: "notifications_settings#unsubscribe_email", as: :unsubscribe_email
-        post "/subscribe_email", to: "notifications_settings#subscribe_email", as: :subscribe_email
```

Comment:
re: pair review, maybe set up temporary redirect for existing links that have been emailed out